src/main/java/com/guichaguri/minimalftp/FTPConnection.java

```diff
@@ -402,6 +420,10 @@ public void registerCommand(String label, String help, ArgsArrayCommand cmd, boo
         addCommand(label, help, cmd, needsAuth);
     }
 
+    public void unregisterCommand(String label) {
```

Comment:
Is there a reason for adding this method? I think `removeCommand` is enough

Reply:
Was just following convention with registerCommand/unregisterCommand & addCommand/removeCommand :)

Reply:
Fair enough